recipe/meta.yaml

```diff
@@ -87,8 +87,8 @@ outputs:
       build:
         - {{ compiler("c") }}
         - {{ compiler("cxx") }}
```

Comment:
Should we provide `pkg-config` as there are pkconfig files added for `linux`?

Reply:
The build scripts are moving files around, but are not calling pkg-config. So it's not required.